README.rst

```diff
@@ -179,7 +179,7 @@ You can find a quick demo of coala here:
 
 |asciicast|
 
-.. |asciicast| image:: https://asciinema.org/a/42968.png
+.. |asciicast| image:: http://i.imgur.com/rLasBoQ.png
    :target: https://asciinema.org/a/42968?autoplay=1
    :width: 100%
```

Comment:
I tried changing the width but then after converting to html (used an online rst to html converter), it has no effect.
And probably pypi first converts it into html to display it on their website and in their website also, width variable was not passed while converting it to html.